src/SerializeAST.cc

```diff
@@ -823,66 +830,88 @@ void findNamedDataStructures(const snowcrash::Elements& elements, DataStructures
 
     }
 }
-#endif
 
-sos::Object drafter::WrapBlueprint(const Blueprint& blueprint, const ASTType astType)
+void registrateNamedMSONStructures(const snowcrash::Elements& elements, refract::Registry& registry) 
```

Comment:
Well, they are called `Named Types` and I think `registrate` is completely different meaning to `register`.